src/main/java/io/spring/githubchangeloggenerator/ChangelogGenerator.java

```diff
@@ -142,14 +148,17 @@ private String generateContent(List<Issue> issues) {
 		return content.toString();
 	}
 
-	private void addSectionContent(StringBuilder content, Map<ChangelogSection, List<Issue>> sectionIssues) {
-		sectionIssues.forEach((section, issues) -> {
-			sort(section.getSort(), issues);
-			content.append((content.length() != 0) ? String.format("%n") : "");
-			content.append("## ").append(section).append(String.format("%n%n"));
-			issues.stream().map(this::getFormattedIssue).forEach(content::append);
-		});
-	}
+    private void addSectionContent(StringBuilder content,
+            Map<ChangelogSection, List<Issue>> sectionIssues) {
+        sectionIssues.forEach((section, issues) -> {
+            sort(section.getSort(), issues);
+            content.append((!content.isEmpty()) ? String.format("%n") : "");
```

Comment:
Switching to `isEmpty()` is unrelated to allowing more control over the formatting. Please revert.